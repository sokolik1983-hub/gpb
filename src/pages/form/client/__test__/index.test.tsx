import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { ClientFormPage } from '..';

describe('ClientFormPage', () => {
  it('renders', () => {
    const wrapper = shallow(<ClientFormPage />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
