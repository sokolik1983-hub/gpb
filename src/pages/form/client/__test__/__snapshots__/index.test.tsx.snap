// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClientFormPage renders 1`] = `
<t>
  Клиентская форма
</t>
`;
